src/components/Toggle/README.md

```diff
@@ -0,0 +1,142 @@
+---
+name: Toggle
+category: Forms
+platforms:
+  - web
+keywords:
+  - toggle
+  - switch
+  - selection
+  - choices
+  - options
+  - pick
+  - single selection form
+  - choice form
+  - option button
+  - enable
+  - disable
+---
+
+# Toggle
+
+Use to present a single choice to enable or disable a setting.
+
+---
+
+## Best practices
+
+Toggle should:
+
+- Always be used with an associated label component.
+- Be used to switch a single setting on and off.
+
+---
+
+## Content guidelines
+
+### Toggle labels
+
+Toggle labels should:
+
+- Start with a capital letter
+
+<!-- usagelist -->
+
+#### Do
+
+- Off
+
+#### Don’t
+
+- off
+
+<!-- end -->
+
+- Not end in punctuation if it’s a single sentence, word, or a fragment
+
+<!-- usagelist -->
+
+#### Do
+
+- Off
+
+#### Don’t
+
+- Off;
+
+<!-- end -->
+
+---
+
+## Examples
+
+### Default toggle
+
+Use toggle to let merchants enable or disable a setting.
+
+```jsx
+class RadioButtonExample extends React.Component {
+  state = {
+    checked: false,
+  };
+
+  toggleToggle = (value) => {
+    this.setState({checked: value});
+  };
+
+  render() {
+    const {value} = this.state;
+    return (
+      <Stack vertical>
+        <Toggle
+          label="Accept email marketing"
+          labelHidden
+          checked={this.state.checked}
+          onChange={this.toggleToggle}
+        />
+      </Stack>
+    );
+  }
+}
+```
+
+### Prefix and suffix elements
+
+<!-- example-for: web -->
+
+Use to provide more clarity to merchants about which setting they are selecting.
+
+```jsx
+class RadioButtonExample extends React.Component {
+  state = {
+    checked: false,
+  };
+
+  toggleToggle = (value) => {
+    this.setState({checked: value});
+  };
+
+  render() {
+    const {value} = this.state;
+    return (
+      <Stack vertical>
+        <Toggle
+          label="Accept email marketing"
+          labelHidden
+          prefix="Off"
+          suffix="On"
+          checked={this.state.checked}
+          onChange={this.toggleToggle}
+        />
+      </Stack>
```

Comment:
This feels like it should be the default state for this component. It's far more accessible and creates a better UX when we provide these labels.

Reply:
I stumbled here because toggle is generally associated with a longer, inline description, rather than a label that's positioned above.